packages/consumption/test/modules/attributes/AttributesController.test.ts

```diff
@@ -2067,19 +2193,39 @@ describe("AttributesController", function () {
                 await consumptionController.attributes.succeedOwnSharedIdentityAttribute(ownSharedIdentityAttributeV1PeerB.id, ownSharedIdentityAttributeParamsV2PeerB));
         });
 
+        test("should return all shared predecessors for all peers", async function () {
```

Comment:
Check the order of returned predecessors / successors, if that's a behavior you want users of your function to rely on.

Reply:
I check for the order per peer, but for the same version shared with multiple peers the order of returns varies.